modules/releasing/pages/tenant-release-pipelines.adoc

```diff
@@ -175,6 +175,9 @@ spec:
 
 If you write a good reusable release pipeline, please submit it to our link:https://github.com/konflux-ci/release-service-catalog[catalog] so others can use it.
```

Comment:
This page is about tenant release pipelines, so I think your new note should be in place of this one, not in addition to. Now that we have community catalog, there is no need to have any tenant/final pipeline resources in release-service-catalog